addons/mail/data/tour.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
:socks: Name the file web_tour_tour.xml (it should match the model inside it so that it is easy to grep files)